### DISCUSSION TITLE
Can't find @alpinejs/csp

### DISCUSSION THREAD

**Discussion by nickurban** [alpinejs/alpine#1529] [7. Bugs (archived)]
Hi, I added alpinejs 3.0.6 to my package.json. This works: `import Alpine from alpinejs`.

However, when I try: `import Alpine from '@alpinejs/csp'` (per the instructions here: https://alpinejs.dev/advanced/csp), I get `Cannot find module '@alpinejs/csp'`

**Comment by antja0** [comment]
any updates on this ? does csp version of alpines even exist?

**Reply by antja0** [reply]
or can you write csp friendly code with the main build?

**Comment by browner12** [comment]
The CSP build has not been deployed to NPM, so the instructions are not entirely accurate.

The only way to currently get the CSP build is clone the repo, run the build, and copy/paste the resulting ESM or CJS file.

Fair warning, I've found the CSP build to be basically unusable. Even something as simple as an `x-show` does not appear to be working.
